Library/Homebrew/dev-cmd/create.rb

```diff
@@ -18,14 +19,16 @@ def create_args
       usage_banner <<~EOS
         `create` [<options>] <URL>
 
-        Generate a formula for the downloadable file at <URL> and open it in the editor.
+        Generate a formula or cask for the downloadable file at <URL> and open it in the editor.
         Homebrew will attempt to automatically derive the formula name and version, but
         if it fails, you'll have to make your own template. The `wget` formula serves as
         a simple example. For the complete API, see:
         <https://rubydoc.brew.sh/Formula>
       EOS
       switch "--autotools",
              description: "Create a basic template for an Autotools-style build."
+      switch "--cask",
+             description: "Create a basic template for a Cask."
```

Comment:
My take is to keep "cask" lowercase except in the context of the module itself, e.g. "list of casks" vs. "Homebrew Cask".

Reply:
I think the usage of “cask” vs. “Cask“ is generally very inconsistent across the code base.